src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/InstanceEventRepository.cs

```diff
@@ -0,0 +1,245 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Configuration;
+using Altinn.Platform.Storage.Models;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.Documents.Linq;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace Altinn.Platform.Storage.Repository
+{
+    /// <summary>
+    /// Handles instanceEvent repository
+    /// </summary>
+    public class InstanceEventRepository : IInstanceEventRepository
+    {
+        private readonly Uri _databaseUri;
+        private readonly Uri _collectionUri;
+        private readonly string databaseId;
+        private readonly string collectionId = "instanceEvent";
```

Comment:
events or instance-events

Reply:
Will be changed to instance-events

Reply:
I'm just worried here to use a hyphen in the collection name as we encountered problems in c# classes because of hyphen. So i would suggest that do a testing of fetching data from this colleciton(instance-events) using plain sql query, built in methods etc to know if that doesn't cause any issues. Also check it out in cosmos db emulator by running queries there @dskogan @acn-sbuad